.gitignore

```diff
@@ -128,3 +128,5 @@ dist
 .yarn/build-state.yml
 .yarn/install-state.gz
 .pnp.*
+
+public/*
```

Comment:
This is totally what you'd usually do, but I think since we have some folks who're so new to this it might be useful to keep this unignored for a little. Just a thought though.

Reply:
Sure, that's cool, just didn't want to bloat the repo. How does it make it easier for newcomers?

Reply:
Just serves as a constant reminder of the transpilation step, which is a bit weird for folks not used to typescript, as somewhat evidenced by Alex's questions lol

Reply:
10-4, works for me!